components/org.wso2.carbon.sp.jobmanager.core/src/gen/java/org/wso2/carbon/sp/jobmanager/core/model/NodeConfig.java

```diff
@@ -149,6 +169,7 @@ public String toString() {
         sb.append("    id: ").append(toIndentedString(id)).append("\n");
         sb.append("    state: ").append(toIndentedString(state)).append("\n");
         sb.append("    httpInterface: ").append(toIndentedString(httpInterface)).append("\n");
```

Comment:
Are we supporting both HTTP and HTTPS or only HTTPS?

Reply:
Only HTTPS